### ISSUE TITLE
Docs: Typo bugs in Chapter 1 and 2

### ISSUE THREAD

**Issue by bsrikan** [marklogic/entity-services#277]
Issue1:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/intro#id_28145
Actual: 
Persistence Convention: The entity pesistence pattern promoted by Entity Services
Expected:
Persistence Convention: The entity persistence pattern promoted by Entity Services

Issue2:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/getting-started#id_84640
Actual:
The example also uses es.modelValidate to checks the descriptor for errors before inserting it.
Expected:
The example also uses es.modelValidate to check the descriptor for errors before inserting it.

Issue3:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/getting-started#id_pgfId-1117445
Actual:
Also, the most application will require converter customizations.
Expected:
Also, most applications will require converter customizations.

Issue4:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/getting-started#id_pgfId-1117467
Actual:
Locate the section of es:extract-instance-Person
Expected:
Locate the section of person:extract-instance-Person

Issue5:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/getting-started#id_41004
Actual:
You create an envelope document for some entity type T using the extract-instance-T and instance-to-envelope functions of the instance converter.
Expected:
You create an envelope document for some entity type T using the extractInstanceT and instanceToEnvelope functions of the instance converter.

Issue6:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/getting-started#id_97021
Actual:
However, the original JSON source data must be converted from a string to a JSON node using xdmp:from-json-string
Expected:
However, the original JSON source data must be converted from a string to a JSON node using xdmp.unquote

Issue7:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/getting-started#id_72815
Actual:
Template Driven Extraction (TDE) template that enables row and semantic search of instances. For details, see .
Expected:
a link after "For details, see". Maybe this is still WIP?

**Comment by bsrikan** [comment]
Kim seems to have pushed fixes for the issues in this bug. Updates are in Bugtrack # 45195. Assigning to self with label test.

**Comment by bsrikan** [comment]
For issue#5 I ask to use sjs version since the example box below is sjs and uses instanceToEnvelope and extractInstanceT.

I am not sure about issue6. Looks like what we have there now(xdmp:from-json-string) is correct. Maybe I switched expected and actual.

**Comment by kcoleman-marklogic** [comment]
No problem with Issue 6. You were correct about the mismatch between the text and the example. I just fixed it in a different way than you expected. Both xdmp:unquote and xdmp:from-json-string work here. I chose to go with xdmp:from-json-string.

I understand why you raised Issue 5. You're not wrong, but as I said in Bugtrack, there's not really a good answer there, IMO. I chose to leave it.
